testing/src/main/java/org/oppia/android/testing/story/StoryProgressTestHelper.kt

```diff
@@ -346,14 +403,67 @@ class StoryProgressTestHelper @Inject constructor(
   }
 
   /**
-   * Marks the only chapter of test topic 1 story 2 as recently played. For specifics on parameters
-   * and nuances, see: [markRecentlyPlayedTestTopic0Story0Exp0].
+   * Marks the second chapter of test topic 0 story 0 as in progress not saved. For specifics on
+   * parameters and nuances, see: [markInProgressNotSavedTestTopic0Story0Exp0].
    */
-  fun markRecentlyPlayedTestTopic1Story2Exp0(
+  fun markInProgressNotSavedTestTopic0Story0Exp1(
     profileId: ProfileId,
     timestampOlderThanOneWeek: Boolean
   ) {
-    recordRecentlyPlayedChapter(
+    // Must complete the previous chapter first.
+    markCompletedTestTopic0Story0Exp0(profileId, timestampOlderThanOneWeek)
+    recordChapterAsInProgressNotSaved(
+      profileId,
+      TEST_TOPIC_ID_0,
+      TEST_STORY_ID_0,
+      TEST_EXPLORATION_ID_5,
+      timestampOlderThanOneWeek
+    )
+  }
+
+  /**
+   * Marks the only chapter of test topic 1 story 2 as started, but yet not completed. For specifics on parameters
+   * and nuances, see: [markStartedNotCompletedTestTopic0Story0Exp0].
+   */
+  fun markStartedNotCompletedTestTopic1Story2Exp0(
```

Comment:
Add description of arguments in kdoc using  `@param `

Reply:
added to markStartedNotCompletedTestTopic1Story2Exp0